application/src/main/java/org/opentripplanner/standalone/config/BuildConfig.java

```diff
@@ -287,6 +293,7 @@ When set to true (it is false by default), the elevation module will include the
           "Transfers up to this duration with the default walk speed value will be pre-calculated and included in the Graph."
         )
         .asDuration(Duration.ofMinutes(30));
+    transferParametersForMode = TransferConfig.map(root, "transferParameters");
```

Comment:
This variable should be renamed if we decide to make the `transferParameters` used for other transfer parameters as well.